c/common/platform.h

```diff
@@ -490,6 +496,11 @@ static BROTLI_INLINE brotli_reg_t BrotliRBit(brotli_reg_t input) {
 static BROTLI_INLINE void BrotliRBit(void) { /* Should break build if used. */ }
 #endif  /* BROTLI_RBIT */
 
+#if ((defined(BROTLI_TARGET_ARMV7) || defined(BROTLI_TARGET_ARMV8_32)) && \
```

Comment:
Sorry for nit-picking. Lets either generalize the name/purpose, or move this piece to `huffman.h` - though it is platform-dependent, it is very narrow - only for huffman tables and only for decoder.